common/lib/xmodule/xmodule/templates/problem/checkboxes_response.yaml

```diff
@@ -21,27 +21,26 @@ metadata:
        Urdu, Marathi, and French are all Indo-European languages, while Finnish and Hungarian are in the Uralic family.
        [explanation]
 
-data: |
-      <problem>
-        <p>Checkbox problems allow learners to select multiple options. Learners can see all the options along with the problem text.</p>
-        <p>When you add the component, be sure to select <strong>Settings</strong> 
-        to specify a <strong>Display Name</strong> and other values that apply.</p>
-        <p>You can use the following example problem as a model.</p>
-          <p>The following languages are in the Indo-European family:</p>
-          <choiceresponse>
-            <checkboxgroup>
-              <choice correct="true" name="urdu">Urdu</choice>
-              <choice correct="false" name="finnish">Finnish</choice>
-              <choice correct="true" name="marathi">Marathi</choice>
-              <choice correct="true" name="french">French</choice>
-              <choice correct="false" name="hungarian">Hungarian</choice>
-            </checkboxgroup>
-          </choiceresponse>
-        <p><strong>Note</strong>: Make sure you select all of the correct options—there may be more than one!</p>
-        <solution>
-         <div class="detailed-solution">
-             <p>Explanation</p>
-             <p>Urdu, Marathi, and French are all Indo-European languages, while Finnish and Hungarian are in the Uralic family.</p> 
-          </div>
-        </solution>
-      </problem>
\ No newline at end of file
+data:   |
+        <problem>
+            <choiceresponse>
+                <p>Checkbox problems allow learners to select multiple options. Learners can see all the options along with the problem text.</p>
+                <p>When you add the problem, be sure to select Settings to specify a Display Name and other values that apply.</p>
+                <p>You can use the following example problem as a model.</p>
+                <label>The following languages are in the Indo-European family:</label>
+                <checkboxgroup label="The following languages are in the Indo-European family:">
+                    <choice correct="true">Urdu</choice>
+                    <choice correct="false">Finnish</choice>
+                    <choice correct="true">Marathi</choice>
+                    <choice correct="true">French</choice>
+                    <choice correct="false">Hungarian</choice>
+                </checkboxgroup>
+                <p>Note: Make sure you select all of the correct options—there may be more than one!</p>
```

Comment:
@cahrens We're going to move this text in question description with "Note:" in upcoming descriptions PR.